app/Models/User.php

```diff
@@ -41,4 +41,10 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    protected $table='users';
+
+    public function notes(){
```

Comment:
```suggestion
    public function notes()
    {
```
các hàm thông thường ntn bỏ dấu ngoặc xuống dòng nhé =)